Formula/ooniprobe.rb

```diff
@@ -3,287 +3,146 @@ class Ooniprobe < Formula
-  end
+  depends_on "go@1.14" => :build
 
   def install
-    # provided by libdnet
-    inreplace "requirements.txt", "pydumbnet", ""
-
-    # force a distutils install
-    inreplace "setup.py", "def run(", "def norun("
-
-    # obey the settings.ini we write
-    inreplace "ooni/settings.py", /(IS_VIRTUALENV = ).*/, "\\1 False"
-
-    (buildpath/"ooni/settings.ini").atomic_write <<~EOS
-      [directories]
-      usr_share = #{pkgshare}
-      var_lib = #{var}/lib/ooni
-      etc = #{etc}/ooni
-    EOS
-
-    ENV.append "CPPFLAGS", "-I#{MacOS.sdk_path}/usr/include/ffi" if MacOS.sdk_path_if_needed
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    (buildpath/"src/github.com/ooni/probe-cli").install buildpath.children
 
-    virtualenv_install_with_resources
+    cd "src/github.com/ooni/probe-cli" do
+      system "go", "build", "-o", bin/"ooniprobe", "./cmd/ooniprobe"
 
-    man1.install Dir["data/*.1"]
-    pkgshare.install Dir["data/*"]
-  end
-
-  def post_install
-    return if (pkgshare/"decks-available").exist?
-
-    ln_s pkgshare/"decks", pkgshare/"decks-available"
-    ln_s pkgshare/"decks/web.yaml", pkgshare/"current.deck"
-  end
+      prefix.install_metafiles
+    end
 
-  def caveats
-    <<~EOS
-      Decks are installed to #{opt_pkgshare}/decks.
+    (pkgshare/"etc/ooniprobe-daily-config.json").write <<~EOS
+    {
+      "_version": 3,
+      "_informed_consent": true,
+      "sharing": {
+        "include_ip": false,
+        "include_asn": true,
+        "upload_results": true
+      },
+      "nettests": {
+        "websites_url_limit": 100,
+        "websites_enabled_category_codes": [
+          "ALDR",
+          "ANON",
+          "COMM",
+          "COMT",
+          "CTRL",
+          "CULTR",
+          "DATE",
+          "ECON",
+          "ENV",
+          "FILE",
+          "GAME",
+          "GMB",
+          "GOVT",
+          "GRP",
+          "HACK",
+          "HATE",
+          "HOST",
+          "HUMR",
+          "IGO",
+          "LGBT",
+          "MILX",
+          "MISC",
+          "MMED",
+          "NEWS",
+          "POLR",
+          "PORN",
+          "PROV",
+          "PUBH",
+          "REL",
+          "SRCH",
+          "XED"
+        ]
+      },
+      "advanced": {
+        "use_domain_fronting": false,
+        "send_crash_reports": true,
+        "collect_usage_stats": true,
+        "collector_url": "",
+        "bouncer_url": "https://bouncer.ooni.io"
+      }
+    }
     EOS
   end
 
-  plist_options startup: "true", manual: "ooniprobe -i #{HOMEBREW_PREFIX}/share/ooniprobe/current.deck"
+  plist_options startup: "true", manual: "ooniprobe --config \"#{HOMEBREW_PREFIX}/etc/ooniprobe-daily-config.json\" run"
 
   def plist
     <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-      <dict>
+    <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this the user's plist? I'm wondering what happens if the user manually runs `ooniprobe` while an automatic test is running? I think there will be conflict with locking the database? Maybe we want to use a specifically different OONI HOME for this kind of use cases?

Reply:
fixed!

Reply:
They use different OONI_HOME directories now.